CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Given a version number `MAJOR.MINOR.PATCH`, we increment the:
 - Add support for formatting specific files and stdin (`terramate fmt [file...]` or `terramate fmt -`).
 - Add `--cloud-status=status` flag to both `terramate run` and `terramate script run`.
 - Add `--cloud-sync-preview` flag to `terramate run` to sync the preview to Terramate Cloud.
+- Add `terramate create --all-terragrunt`. This allows for an easy onboarding of Terragrunt projects.
```

Comment:
@mariux @soerenmartius 
What else can be said here?

Reply:
How about this?
```
- Improve user experience when using Terramate with existing Terragrunt projects.
  - Add  `terramate create --all-terragrunt` option, which will automatically create Terramate stacks for each Terraform module.
```